src/components/VCarousel/VCarousel.js

```diff
@@ -180,7 +184,10 @@ export default {
           left: this.next,
           right: this.prev
         }
-      }]
+      }],
+      style: {
+        height: !isNaN(this.height) ? `${this.height}px` : this.height
```

Comment:
use `convertToUnit()` from src/util/helpers.ts

Reply:
It's not working well with ``convertToUnit()`` because if height is not given the ``style.height`` must be empty in order to use the default height defined in the style. 
But, with the current ``convertToUnit``, i can't have that result, because :
``convertToUnit('') = '0px'``
``convertToUnit(null) = '0px' ``
``convertToUnit(undefined) = '0px'``
Are these current results expected ?

Reply:
maybe `this.height ? converToUnit(this.height) : undefined`?